File: Praktikum Pemrograman 1/Praktikum Pemrograman 1 modul 5/Python/PRAK504-2410817110006-HamkaArifani.py
def reverse(nilai:int):
    hasil=0
    while nilai>0:
        panjang=nilai % 10
        hasil=hasil * 10 + panjang
        nilai=nilai//10
    return hasil
A,B=map(int,input("").split())
A= reverse(A)
B= reverse (B)
C=A+B
print(reverse(C))

File: Praktikum Pemrograman 1/Praktikum Pemrograman 1 modul 5/C/PRAK504-2410817110006-HamkaArifani.c
#include <stdio.h>
int reverse(int nilai){
    int hasil=0;
    while(nilai>0){
        int panjang=nilai % 10;
        hasil=hasil * 10 + panjang;
        nilai=nilai/10;
        } 
        return hasil;
}
int main() {
int A, B;
scanf("%d %d",&A,&B);
A=reverse(A);
B=reverse(B);
int C = A+B;
printf("%d",reverse(C));
}